starfish/image/_stack.py

```diff
@@ -1,54 +1,47 @@
-import json
 import os
 
 import numpy
+from slicedimage import Reader, Writer
 
-from ._base import ImageBase, ImageFormat
+from ._base import ImageBase
 
 
 class ImageStack(ImageBase):
-    def __init__(self, data, tile_format, num_hybs, num_chs, tile_shape):
-        self._data = data
-        self._tile_format = tile_format
+    def __init__(self, slicedimage):
```

Comment:
use of `slicedimage` here is a bit confusing since it's also the name of the package. I went and looked, I guess this should be `image_partition`, instead?